torch/_dynamo/variables/torch_function.py

```diff
@@ -45,6 +46,19 @@ def is_torch_function_user_object(obj):
     return hasattr(obj, "__torch_function__")
 
 
+def _is_attr_overidden(tx, var, name):
+    import torch
+
+    overridden = False
+    try:
+        attr_val = inspect.getattr_static(var.python_type(), name)
+        overridden |= attr_val != getattr(torch.Tensor, name)
+    except AttributeError:
+        pass
```

Comment:
When do we hit this case?   Is returning false correct?

Reply:
Basically if a class overrides a property or method, `attr_val != getattr(torch.Tensor, name)` will be True, but if the class has the standard `torch.Tensor` implementation this will be False. I added test cases [here](https://github.com/pytorch/pytorch/pull/111737/files#:~:text=def%20test_user_overidden_attr_unsupported(self)%3A) to verify that this works.

Reply:
Instead of catching AttributeError, you could do getattr(torch.Tensor, name, None)